gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -381,14 +479,37 @@ def create_variant_qc_annotation_ht(
     :param n_partitions: Number of partitions to use for final annotated Table.
     :return: Hail Table with all annotations needed for variant QC.
     """
+    info_methods = [f"{m}_info" for m in INFO_METHODS]
     truth_data_ht = get_truth_ht()
 
-    ht = info_ht.transmute(**info_ht.info, **info_ht.allele_info)
+    ht = get_reformatted_info_fields(info_ht)
+    ht = ht.annotate(**{m: ht[m].select(*INFO_FEATURES) for m in info_methods})
+    ht = ht.transmute(**ht.AC_info, **ht.allele_info, **ht.site_info)
+
+    if impute_features:
+        feature_imputed = {}
+        feature_medians = {}
+        for m in info_methods:
+            m_info_ht = ht.select("variant_type", **ht[m])
+            m_info_ht = median_impute_features(
```

Comment:
oops, I forgot to actually annotate ht with the imputed values...will add tomorrow